appinventor/components/src/com/google/appinventor/components/common/UltrasonicSensorMode.java

```diff
@@ -0,0 +1,46 @@
+// -*- mode: java; c-basic-offset: 2; -*-
+// Copyright 2020 MIT, All rights reserved
+// Released under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+package com.google.appinventor.components.common;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Defines a UltrasonicSensorMode type used by the Lego Ev3 ultrasonic sensor.
+ */
+public enum UltrasonicSensorMode implements OptionList<String> {
```

Comment:
It seems unintuitive to me that we plug a block called `UltrasonicSensorMode` into a property called `Unit`. I think we should rename this to `UltrasonicSensorUnit` instead to match the corresponding property.